content-src/experiments/containers.yaml

```diff
@@ -58,7 +58,7 @@ tour_steps:
     copy: Start a Container to label and organize a set of tabs.
   -
     image: /static/images/experiments/containers/tour/tour2.jpg
-    copy: Hover over the New Tab button to select a container.
+    copy: Long press on the New Tab button to select a container.
```

Comment:
This looks exactly right! r+ from me, we'll wait to merge this until it's reviewed by our head localizer, who reviews all string changes.